packages/seed-bible/components/component/components.component.bot.aux

```diff
@@ -1,12 +1,12 @@
ion:ease-in-out}@keyframes drop-140{50%{top:2%;left:50%}100%{top:110%;left:55%}}.confetti-141{width:1px;height:.4px;background-color:#ef3d4d;top:150%;left:58%;opacity:.9438828528;transform:rotate(82.5701318882deg);animation:drop-141 3.5629505909s .1172828324s infinite;animation-timing-function:ease-in-out}@keyframes drop-141{50%{top:5%;left:68%}100%{top:110%;left:61%}}.confetti-142{width:6px;height:2.4px;background-color:#7d3af2;top:150%;left:52%;opacity:.7710009818;transform:rotate(184.0555450239deg);animation:drop-142 3.8615173227s .9531579811s infinite;animation-timing-function:ease-in-out}@keyframes drop-142{50%{top:10%;left:62%}100%{top:110%;left:64%}}.confetti-143{width:5px;height:2px;background-color:#2971f9;top:150%;left:91%;opacity:1.2386292928;transform:rotate(356.9582223361deg);animation:drop-143 3.9412474261s .6132379257s infinite;animation-timing-function:ease-in-out}@keyframes drop-143{50%{top:7%;left:99%}100%{top:110%;left:97%}}.confetti-144{width:2px;height:.8px;background-color:#2971f9;top:150%;left:57%;opacity:.799633153;transform:rotate(121.0003729923deg);animation:drop-144 3.3456715656s .9672150584s infinite;animation-timing-function:ease-in-out}@keyframes drop-144{50%{top:3%;left:66%}100%{top:110%;left:62%}}.confetti-145{width:8px;height:3.2px;background-color:#f9ca16;top:150%;left:29%;opacity:1.407763382;transform:rotate(316.1906750563deg);animation:drop-145 3.8736001063s 56.7429298ms infinite;animation-timing-function:ease-in-out}@keyframes drop-145{50%{top:2%;left:30%}100%{top:110%;left:43%}}.confetti-146{width:2px;height:.8px;background-color:#2971f9;top:150%;left:47%;opacity:1.2394303955;transform:rotate(.7540497084deg);animation:drop-146 3.4553683742s .93501134s infinite;animation-timing-function:ease-in-out}@keyframes drop-146{50%{top:1%;left:49%}100%{top:110%;left:49%}}.confetti-147{width:4px;height:1.6px;background-color:#ef3d4d;top:150%;left:71%;opacity:.997744401;transform:rotate(233.953330119deg);animation:drop-147 3.1643660009s .3888895436s infinite;animation-timing-function:ease-in-out}@keyframes drop-147{50%{top:10%;left:86%}100%{top:110%;left:83%}}.confetti-148{width:1px;height:.4px;background-color:#f9ca16;top:150%;left:20%;opacity:.635149758;transform:rotate(310.8837498087deg);animation:drop-148 3.2775241996s .8955929872s infinite;animation-timing-function:ease-in-out}@keyframes drop-148{50%{top:8%;left:31%}100%{top:110%;left:30%}}.confetti-149{width:2px;height:.8px;background-color:#f9ca16;top:150%;left:65%;opacity:1.4859330578;transform:rotate(169.351202976deg);animation:drop-149 3.5422889063s .9414184111s infinite;animation-timing-function:ease-in-out}@keyframes drop-149{50%{top:10%;left:69%}100%{top:110%;left:68%}}.confetti-150{width:1px;height:.4px;background-color:#7d3af2;top:150%;left:32%;opacity:1.0784230768;transform:rotate(178.2288931143deg);animation:drop-150 3.0670691708s .9202188868s infinite;animation-timing-function:ease-in-out}@keyframes drop-150{50%{top:8%;left:47%}100%{top:110%;left:40%}}",
```

Comment:
@Kabi4 is this tag used? It looks like a giant blob of minified CSS, is that expected here?